docs/building/nocore-ebpf.md

```diff
@@ -60,26 +67,27 @@ embedded CO-RE eBPF object, as a last resource if there is no:
 
 1. BTF file available in running kernel (`/sys/kernel/btf/vmlinux`).
 1. BTF file pointed by `TRACEE_BTF_FILE` environment variable.
-1. BTF file embedded into "tracee-ebpf" binary ([BTFhub](https://github.com/aquasecurity/btfhub)).
+1. BTF file embedded into "tracee-ebpf" binary ([BTFHUB](https://github.com/aquasecurity/btfhub)).
 
-> **NOTE**: Installing the non CO-RE eBPF object does not mean you will run
-> `tracee-ebpf` with it by default. It only means that if your running kernel
-> doesn't support CO-RE, it will be loaded. To force `tracee-ebpf` to always
-> run non CO-RE eBPF object you have to specific TRACEE_BPF_FILE= environment
-> variable.
+!!! Note
+    Installing the non CO-RE eBPF object does not mean you will run
+    **tracee-ebpf** with it by default. It only means that if your running
+    kernel doesn't support CO-RE, it will be loaded. To force **tracee-ebpf** to
+    always run non CO-RE eBPF object you have to specific TRACEE_BPF_FILE=
+    environment variable.
```

Comment:
Is this still true?
Quoting from main.go:

> 	// Decision ordering:
> 
> 	// (1) BPF file given & BTF (vmlinux or env) exists: always load BPF as CO-RE
> 	// (2) BPF file given & if no BTF exists: it is a non CO-RE BPF

Reply:
Changed it to make it more clear:

```
!!! Note
    Installing the non CO-RE eBPF object in the environment does not mean will
    will run **tracee-ebpf** with it by default. If your system supports CO-RE
    eBPF objects it will be chosen instead. If your system supports CO-RE eBPF
    but does not contain embedded BTF information, but is support by BTFHUB,
    then the CO-RE eBPF object will be used by default. The only way you can
    make sure the non CO-RE eBPF object is used is by always informing the
    `TRACEE_BPF_FILE=...` environment variable.
```